ads/vendor-status.md

```diff
@@ -0,0 +1,29 @@
+## AdTech Vendor Integration Status
+There are a number of adtech vendors that are actively integrating with AMP. If you don't find your ad tech provider listed below, we encourage you to invite the vendor directly to create first class support in AMP where it makes sense.
+If you are adtech vendor and would like to provide AMP support, please start by reading the ads [integration guide](https://github.com/ampproject/amphtml/blob/master/ads/integration-guide.md). 
```

Comment:
Can you add a note in the integration guide that adtech vendors should add themselves here?